docs/docs/release-notes/infrahub/release-1_0_8.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Release 1.0.8 - DEVELOPMENT
```

Comment:
Should the `DEVELOPMENT` part be here?

Reply:
Good catch! I am going to go ahead and remove it from the upstream template as well, because it doesn't seem to really add value since the file is already named draft at that point in time.